### ISSUE TITLE
Revisit Arrow and Bolt Recipes

### ISSUE THREAD

**Issue by Bakelt** [D2R-Reimagined/d2r-reimagined-mod#353]
There are two sets of arrow/bolt recipes from different imported mods.

One produces white arrows with various effects on hits (curses). These are too strong in what they provide and with the CtC rate.
The other produces three tiers of arrows that slowly improve with moderate effects, similar to splash charm, but aren't directly linked.

We should settle on one method and remove the other.

My preference is that we make it similar to the progressive splash charm recipe. Could also have a few recipes at the final tier that allows for a selection of CtC skills at a high end game cost.

**Comment by Delegus** [comment]
Yeah, would prefer to go with your second option for this also personally.

**Comment by Delegus** [comment]
Updated and fixed in Pull #456
